exercises/pig-latin/description.md

```diff
@@ -5,10 +5,12 @@ confusing. It obeys a few simple rules (below), but when it's spoken
 quickly it's really difficult for non-children (and non-native speakers)
 to understand.
 
-- **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to
-  the end of the word.
-- **Rule 2**: If a word begins with a consonant sound, move it to the
+* **Rule 1**: If a word begins with a vowel sound, add an "ay" sound to
+  the end of the word. Treat "xr" and "yt" at the beginning of a word as vowels ("xray" -> "xrayay").
+* **Rule 2**: If a word begins with a consonant sound, move it to the
   end of the word, and then add an "ay" sound to the end of the word.
+* **Rule 3**: If a word starts with the following consonant clusters "ch", "qu", "th", "thr", "sch" and any consonant followed by "qu" at the beginning of the word, move it to the end of the word, and then add an "ay" sound to the end of the word (e.g. "chair" -> "airchay", "square" -> "aresquay", "thread" -> "eadthray").
+* **Rule 4**: If "y" comes as the beginning of the word then it must be treated as a consonant, and if it comes after a consonant cluster or is the second letter in a two letter word it must be treated as a vowel.
```

Comment:
This rule appears to conflict with the added portion of Rule 1. If the intent is to make a maximally explicit ruleset, that'll need some clarification.

Reply:
Are you referring to Rule #4. Do you think I should integrate it into Rule #1?

Reply:
Yes, I'm referring to rule 4. I don't believe it needs to be integrated, but it does need to be deconflicted.